src/nfa/nfa.cc

```diff
@@ -118,21 +116,21 @@ namespace {
     /**
      * Add transitions to the trimmed automaton.
      * @param[in] nfa NFA to add transitions from.
-     * @param[in] original_to_new_states_map Map of old states to new trimmed automaton states.
+     * @param[in] original_to_new_state_renaming Map of old states to new trimmed automaton states.
```

Comment:
Again, long name. 
Maybe we can use the fact that these things are always 
some_parameter_nfa -> resulting_nfa renaming, and use short names to indicate what is what if needed.
Here only state_renaming.
In the case of a function with lhs and rhs parameter, it could be lhs_state_renaming, rhs_state_renaming.
??

Reply:
Just by the way, not relevant to this PR, rename compute_reachability to something as
reachable_from, states_reachable_from, reachable, reachable_states
??
And instead two version of the function, it could be one with an optional parameter?

Reply:
>  In the case of a function with lhs and rhs parameter, it could be lhs_state_renaming, rhs_state_renaming.

I second your idea. These were automatic search-and-replace modifications. The purpose of this PR is not to refactor naming of function parameters. This will be resolved in a later PR refactoring input and output parameters.

Reply:
>  And instead two version of the function, it could be one with an optional parameter?

Both functions `compute_reachability()` are functions in anonymous namespace, meaning they only exist in the `nfa.cc` source file, and cannot be called outside this file. The function in the Mata interface is `get_reachable_states()` which calls `compute_reachability()`. We can see what could be done about refactoring these two functions.

Reply:
On second thought, I will rename the variables in this PR already.

Reply:
Both functions have been merged into one.